src/uci.cpp

```diff
@@ -169,6 +171,68 @@ namespace {
     Threads.start_thinking(pos, States, limits);
   }
 
+  // Extract the next move from the TT
+
+  void extract_move_from_tt(Position& pos, istringstream& is) {
+
+    Move m;
+    string token, fen;
+    vector<Move> moves;
+
+    StateListPtr pvMovesStates(new std::deque<StateInfo>(1));
+    if (is >> token && token == "moves") {
+        // Parse move list (if any)
+        while (is >> token && (m = UCI::to_move(pos, token)) != MOVE_NONE)
+        {
+            if (MoveList<LEGAL>(pos).contains(m)) {
+                pvMovesStates->push_back(StateInfo());
+                pos.do_move(m, pvMovesStates->back());
+                moves.push_back(m);
+            } else {
+                //TODO: do what!?
+            }
+        }
+    }
+    bool ttHit = false;
+    TTEntry* tte = TT.probe(pos.key(), ttHit);
+    if (ttHit) {
+        m = tte->move(); // Local copy to be SMP safe
+
+        Depth d = tte->depth();
+        // TODO: this seems swap between +/- need to figure
+        //       out how to make that consistent, right?
+        Value v = tte->value();
+
+        // TODO: not sure what this does.
+        /*bool tb = TB::RootInTB && abs(v) < VALUE_MATE - MAX_PLY;
```

Comment:
I think if the root position is in the syzygybase, this looks up the win/draw/loss evaluation.